capstone/capapi/serializers.py

```diff
@@ -167,6 +169,15 @@ def as_dict(obj):
                 extracted_cite['pin_cites'] = c['pin_cites']
             extracted_citations.append(extracted_cite)
 
+        # move head_matter outside of casebody_data
+        head_matter = list(filter(lambda x: x['type'] == 'head_matter', s['casebody_data']['text']['opinions']))
+        head_matter = head_matter[0] if head_matter else []
+        if head_matter:
+            s['casebody_data']['text']['opinions'].remove(head_matter)
+
+        if 'text' in head_matter:
```

Comment:
Is this `if 'text' in head_matter:` block right? I would think it would want to be `s['casebody_data']['text']['head_matter'] = head_matter['text']` within the `if head_matter:` block, but I could be misreading.